tools/parse/src/main/scala/org/allenai/nlpstack/parse/PolytreeParser.scala

```diff
@@ -1,31 +1,44 @@
 package org.allenai.nlpstack.parse
 
+import java.io.File
+
 import org.allenai.datastore.Datastores
 import org.allenai.nlpstack.core.DependencyParser
 import org.allenai.nlpstack.core.graph.Graph
 import org.allenai.nlpstack.core.parse.graph.{ DependencyGraph, DependencyNode }
 import org.allenai.nlpstack.core.PostaggedToken
 import org.allenai.nlpstack.parse.poly.polyparser
+import org.allenai.nlpstack.parse.poly.polyparser.{ MultiPolytreeParseSource, ConllX, FileBasedPolytreeParseSource }
 
 /** Wrapper for the polyparser using the DependencyParser interface.
   *
   * @param modelFile filename for the parser model
   * @param modelVersion version of the parser model
   * @param useLocalFile if false, then the model file is found on the datastore
+  * @param cacheFiles a sequence of CoNLL files containing "gold" parses to cache
   */
 class PolytreeParser(
```

Comment:
I added some comments to the API to describe how to specify a cache more clearly.